frontend/src/components/MakerForm/MakerForm.tsx

```diff
@@ -613,11 +614,6 @@ const MakerForm = ({
         }}
         zoom={maker.latitude != null && maker.longitude != null ? 6 : undefined}
       />
-      <Collapse in={Object.keys(limits).lenght === 0}>
-        <div style={{ display: Object.keys(limits) === 0 ? '' : 'none' }}>
-          <LinearProgress />
-        </div>
-      </Collapse>
```

Comment:
I removed the linear progress in favor of the circular progress in coordinator's avatar